std/typecons.d

```diff
@@ -163,8 +163,9 @@ public:
         debug(Unique) writeln("return from Release");
         return u;
     }
+
     /** Forwards member access to contents. */
-    auto opDot() inout { return _p; }
+    mixin Proxy!_p;
```

Comment:
Pretty sure Proxy can only be used with lvalues, but p can be an lvalue or an rvalue